airbyte-webapp/src/packages/cloud/views/DefaultView.tsx

```diff
@@ -17,6 +27,11 @@ export const DefaultView: React.FC = () => {
           : `/${RoutePaths.Workspaces}/${workspaces[0].workspaceId}`
       }
       replace
+      // exp-signup-selected-source-definition
+      {...(isSignupSourceSelectorExperiment && {
+        state: { sourceDefinitionId },
+        to: `/${RoutePaths.Workspaces}/${workspaces[0].workspaceId}/${RoutePaths.Source}/${RoutePaths.SourceNew}`,
```

Comment:
I think we lower the UX for new signups significantly with this change. So far users usually run through the New connection or onboarding setup, which basically allows them to setup the source, then destination, then connection, i.e. they end their first flow with a fully setup connection. With this change they'll be navigated to the new source setup page and once they setup the source, will just be on this source's detail page, and basically now anyhow in a flow to setup a destination or create a connection. I think this will decrease our setup experience significantly, and don't think this is a change we should do.

Is there a specific reason we would not want to throw them into the new connection creation flow here and preselect the source?

Reply:
That's a great point actually. I agree it makes more sense to direct them to the Source Setup step on the onboarding flow. I will make the changes.
By the way, any conclusions on how the `hideOnboarding` experiment worked? It's finished?
By your comment, I guess we are keeping the onboarding flow, that way we can reuse it for this one.

Reply:
I'd recommend you throw them into connection setup and not the onboarding flow. We might remove the onboarding page before this experiment here is done, given the low effect we see of it.

Reply:
@timroes this is now updated 💪🏼  https://www.loom.com/share/2f34ee77a49348c3950988b72da02e55